modules/nw-metallb-frr-k8s-merge-multiple-configurations.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-frr-k8s.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-metallb-frr-k8s-merge-multiple-configurations_{context}"]
+= How FRR-K8s merges multiple configurations
+
+In a case where multiple users add configurations that select the same node, `FRR-K8s` merges the configurations.
+Each configuration can only extend others. 
+This means that it is possible to add a new neighbor to a router, or to advertise an additional prefix to a neighbor, but not possible to remove a component added by another configuration.
+
+[id="nw-metallb-frr-k8s-merge-multiple-configuration-conflicts_{context}"]
+== Configuration conflicts
+
+Certain configurations can cause conflicts, leading to errors, for example:
+
+* different ASN for the same router (in the same VRF)
+* different ASN for the same neighbor (with the same IP / port)
+* multiple BFD profiles with the same name but different values
+
+When the daemon finds an invalid configuration for a node, it reports the configuration as invalid and reverts to the previous valid `FRR` configuration.
+
+[id="nw-metallb-frr-k8s-merge-multiple-configurations-merging_{context}"]
+== Merging
+
+When merging, it is possible to:
```

Comment:
Sentence should be complete: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-colons#introductory-text